hack/build.sh

```diff
@@ -9,6 +9,22 @@ declare -a VERSIONS=(2.4)
 OS=$1
 VERSION=$2
 
+function squash {	
+	base=$(grep ^FROM Dockerfile | awk '{print $2}')
+	# install the docker layer squashing tool
+	easy_install --user pip
+	rm -rf docker-scripts
+	git clone --branch 0.3.0 https://github.com/goldmann/docker-scripts/
+	pushd docker-scripts
+	~/.local/bin/pip install --force --user .
+	popd
+	# find the top of the "centos" or "rhel" layer
+	layer=$(~/.local/bin/docker-scripts layers -m ${base} | tail -n 1)
+	# squash everything above it
+	$HOME/.local/bin/docker-scripts squash -f $layer ${IMAGE_NAME}
```

Comment:
Yah, it could be really anything, I let Docker decode it and when it fails, this means that something is wrong with it.